### ISSUE TITLE
when I provide images with a public access link. However, when viewing on mobile, there is no option to download :(

### ISSUE THREAD

**Issue by Toanescphotographer** [immich-app/immich#8802]
### The bug

when I provide images with a public access link. However, when viewing on mobile, there is no option to download :(

https://github.com/immich-app/immich/assets/160454879/4171cb82-6479-40cf-a043-be59431a0f75

### The OS that Immich Server is running on

TrueNAS-SCALE-23.10.1

### Version of Immich Server

1.100.0

### Version of Immich Mobile App

1.100.0

### Platform with the issue

- [ ] Server
- [x] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
i'm using default config on truenas
```

### Your .env content

```Shell
i'm using default config on truenas
```

### Reproduction steps

```bash
1.
2.
3.
...
```

**Comment by alextran1502** [comment]
this was fixed in v1.101.0, update your server and it should work

**Comment by Toanescphotographer** [comment]
> this was fixed in v1.101.0, update your server and it should work

Thank bro.
